wordpress-mobile/WordPress-iOS
# Auth feature flags

Fixes #n/a
WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/179

This does the following:
- Removes the `signInWithApple` feature flag, and moves that logic to `initializeWordPressAuthenticator`.
- Adds a feature flag for the impending `unifiedAuth` project.


To test:
- There should be no change with SIWA, so just verify SIWA is available via Login.
- `unifiedAuth` does nothing yet, so nothing to test.

PR submission checklist:
